config/me/config.go

```diff
@@ -19,10 +19,6 @@ func Configure(p *config.Provider) {
 			Type: "github.com/edixos/provider-ovh/apis/me/v1alpha1.Group",
 		}
 	})
-	p.AddResourceConfigurator("ovh_me_ssh_key", func(r *config.Resource) {
-		r.ShortGroup = shortGroup
-		r.Kind = "SSHKey"
-	})
```

Comment:
@ekarlso is there any reason to delete this ResourceConfigurator?

Reply:
Yes, SSHKey has been deprecated from the updated terraform providers but for some reason referenced still.